SimPEG/NSEM/Utils/testUtils.py

```diff
@@ -0,0 +1,198 @@
+import unittest
```

Comment:
I realize that I did this with the fdem code and am about to recommend against it... so no worries if you don't want to address it in this pr, we can make an issue and deal with it in both the FDEM, NSEM codes in a separate pr. I would suggest putting this code in the tests/EM/NSEM directory. When other people are diving into tests and looking for this code, it is a lot easier if it is close to the tests.